padertorch/data/fragmenter.py

```diff
@@ -107,3 +119,190 @@ def fragment(key, x):
             fragment = deflatten(fragment)
             fragments.append(fragment)
         return fragments
+
+
+def get_anchor(
+        num_samples: int, length: int, shift: int=None,
+        mode: str = 'left', rng=np.random
+):
+    """
+    Calculates anchor for the boundaries for segmentation of a signal
+    with length `num_sammples` in case of a fixed segment length
+    `length` and shift `shift`
+
+    Args:
+        num_samples: num samples of signal for which boundaries are caclulated
+        length: segment length
+        shift: shift between segments, defaults to length
+        mode: Defines the position of the boundaries in the signal:
+            left: anchor is set to zero
+                so that only values at the end are cut
+            right: anchor is set to `num_samples`
+                so that only values at the beginning are cut
+            center: anchor is set to `num_samples // 2`
+            centered_cutout: the anchor is chosen such that the same number
+                of samples are discarded at the end and the beginning
+            random: anchor is set to a random value between
+                `0` and `num_samples`.
+                 This may reduce the number of possible segments.
+            random_max_segments: Randomly chooses the anchor such that
```

Comment:
Given the name `centered_cutout`, would then the name `random_cutout` better match than `random_max_segments`?